src/blocks/networkmanager.rs

```diff
@@ -736,27 +727,19 @@ impl Block for NetworkManager {
 
                                 let ap = if let Ok(ap) = device.active_access_point(&self.dbus_conn)
                                 {
-                                    let ssid = match ap.ssid(&self.dbus_conn) {
-                                        Ok(ssid) => {
-                                            let mut truncated = ssid.to_string();
-                                            truncated.truncate(self.max_ssid_width);
-                                            truncated
-                                        }
-                                        Err(_) => "".to_string(),
-                                    };
-                                    let strength = match ap.strength(&self.dbus_conn) {
-                                        Ok(v) => format!("{}", v).to_string(),
-                                        Err(_) => "0".to_string(),
-                                    };
+                                    let ssid = ap.ssid(&self.dbus_conn).unwrap_or_else(|_| "N/A".to_string());
```

Comment:
So now this displays "N/A" instead of just an empty string? Is the change documented somewhere?